### PR TITLE
[IMP] theme_*: review s_intro_pill occurrences in design themes

### PR DESCRIPTION
**Pull request by chgo-odoo**
*: anelusia, artists, avantgarde, aviato, beauty, bewise, bistro,
bookstore, buzzy clean, cobalt, enark, graphene, kea, kiddo, loftspace,
monglia, nano, notes, odoo_experts, orchid, paptic, real_estate,
treehouse, vehicle, yes, zap

This PR reviews the occurrence of the `s_intro_pill` snippet across design themes. 

- requires : https://github.com/odoo/odoo/pull/175615

task-4077581
Part of task-4077427

### CODE REVIEW

**Comment by anso-odoo** [comment]
I pushed some changes to:
- Rename some sections with old snippet name
- Remove some .DS_Store files that were pushed accidentally

There are conflicts but I'll solve them once the PR is approved by website 👍

File: theme_yes/views/images.xml [hunk 1]
```diff
@@ -284,5 +284,16 @@ Check in theme_kea's primary_variables.scss, theme.scss and theme_common's mixin
...
+<!-- // Intro Pill // -->
+<record id="s_intro_pill_default_image" model="theme.ir.attachment">
+    <field name="key">website.s_intro_pill_default_image</field>
+    <field name="name">website.s_intro_pill_default_image</field>
+    <field name="url">/theme_vehicle/static/src/img/snippets/s_intro_pill_default_image.jpg</field>
+</record>
+<record id="s_intro_pill_default_image_2" model="theme.ir.attachment">
+    <field name="key">website.s_intro_pill_default_image_2</field>
+    <field name="name">website.s_intro_pill_default_image_2</field>
+    <field name="url">/theme_vehicle/static/src/img/snippets/s_intro_pill_default_image_2.jpg</field>
```
Comment (bso-odoo) [inline theme_yes/views/images.xml:296]: Errr... strange car pictures for weddings ^_^; It seems this copy/paste was not adapted.
I noticed it by pure chance - I'll see to add a test to verify we have no such accident in other blocks.

Comment (anso-odoo) [reply theme_yes/views/images.xml]: Well spotted, I double checked the rest and it should be fine now

**Comment by bso-odoo** [comment]
@robodoo delegate=bvr-odoo

**Comment by bvr-odoo** [comment]
@robodoo r+

**Pull request closed without merging**